main.c

```diff
@@ -666,6 +848,7 @@ int main(int argc, char *argv[]) {
 	wl_event_loop_add_signal(state.event_loop, SIGINT, handle_signal, NULL);
 	wl_event_loop_add_signal(state.event_loop, SIGTERM, handle_signal, NULL);
 	wl_event_loop_add_signal(state.event_loop, SIGUSR1, handle_signal, NULL);
+	wl_event_loop_add_signal(state.event_loop, SIGUSR2, handle_signal, NULL);
```

Comment:
I'd prefer this to be discussed in a separate pull request. It's not yet clear whether USR2 should toggle, turn off idle timers, or not exist at all.

Reply:
Rebase and subsequent commits have removed SIGUSR2 from this branch.